src/sagas/dependency.saga.test.js

```diff
@@ -423,9 +425,9 @@ describe('Dependency sagas', () => {
           handleQueueActionCompleted
         )
       );
-      expect(saga.next().value).toEqual(
-        takeEvery(START_NEXT_ACTION_IN_QUEUE, handleStartNextActionInQueue)
-      );
+      // expect(saga.next().value).toEqual( // Todo: Create queue.saga.test file
```

Comment:
@superhawk610 I think we need to create a `queue.saga.test` and move these lines there. Also the other queue related Saga tests should be moved there. 
Is it OK to address this after this PR?

Reply:
Good catch!